index.js

```diff
@@ -48,8 +49,8 @@ var complete = function(phase) {
     return Promise.resolve();
   }
 
-  return phase
-    .reportTest()
+  return Promise
```

Comment:
Even though Raptor itself doesn't rely on the Promise, that's not to say that we never will, or someone else may not. The Promise just represents the fact that this method has a well-defined interface, and it can continue to have that whether it does async work or not.